README.md

```diff
@@ -15,6 +15,17 @@ go get github.com/nats-io/nats.go/
 go get github.com/nats-io/nats-server
 ```
 
+When using Go modules (`GO111MODULE=on`):
+
+```bash
+# Go client latest and explicit version
+go get github.com/nats-io/nats.go/@latest
+go get github.com/nats-io/nats.go/@v1.8.1
+
+# Server
+go get github.com/nats-io/nats-server/v2
```

Comment:
They may need or want the 1.x version so we should show an option for getting that version too, but comment it out and give a comment.

Reply:
thanks, ok made the updates